solutions/dedicated/README.md

```diff
@@ -0,0 +1,60 @@
+# Deployable Architecture (DA) for the dedicated IBM Cloudant instance
+
+Deployable Architecture for the dedicated IBM Cloudant instance which deploys the following resources:
+
+- A resource group, if one is not passed in.
+- A dedicated cloudant instance
```

Comment:
An IBM Cloudant Dedicated instance in IBM Cloud Dedicated (seems strange, but thats what Cloudant [docs](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-ibm-cloud-dedicated) have). @SirSpidey can advise